newsletter2022.html

```diff
@@ -0,0 +1,601 @@
+									<div class="inner">
+										<header class="major">
+											<h2> Academic Development
+											</h2>
+										</header>
+										<p>
+											Scholarship remains an important facet of our brotherhood. 
+											Being in an Engineering and Computer Science fraternity offers our brothers a unique opportunity to study alongside their closest friends.
+											Despite the challenge and rigor of an Engineering or Computer Science course load, brothers earned an average GPA of 3.38, outperforming the UMD average of 3.35 and the all-fraternity average of 3.26, and placing us as the third highest GPA for an IFC fraternity. 
+											In the fall, our Tau pledge class had the second chapter GPA of 3.4, while being among the largest pledge classes in the IFC. 
+											A special shoutout goes to brother Matthew Patrick, who earned his Master's Degree in Human Computer Interaction this Spring.
+										</p>
+									</div>
+								</div>
+							</section>
+
+
+							<section>
+								<a class="image">
+									<img src="images/Newsletter/jobs.png" alt="" data-position="center center" />
+								</a>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Professional Development
+											</h2>
+										</header>
+										<p>
+											In addition to working to find academic success, our chapter places a premium on Professional Development. We have run a number of events designed to help brothers find professional opportunities that suit them, and have continued to work to collect professional resources. 
+											Brothers this semester found continued professional success, and will be working and interning all across the country next year. 
+											In addition to many of us staying in the DC/Maryland/Virginia (DMV) area, multiple graduating brothers will be working in New York and Seattle, and some will be in Boston, San Antonio, and Madison. 
+											Younger brothers have also found plenty of their own success. This summer, in addition to the DMV area, Sig Phi will also be represented in San Francisco, Huntsville, Menlo Park, and Boston, among other places. 
+										</p>
+									</div>
+								</div>
+							</section>
+
+
+							
+							<section>
+								<div class="content">
+									<div class="inner">
+
+									</div>
+								</div>
+							</section>
+
+
+							<section>
+								<a class="image">
+									<img src="https://scontent-iad3-2.xx.fbcdn.net/v/t39.8562-6/148701539_3724512380920015_7768163355512350905_n.png?_nc_cat=110&ccb=1-5&_nc_sid=6825c5&_nc_ohc=65ZhLvsB004AX9D4LJM&_nc_ht=scontent-iad3-2.xx&oh=00_AT8ayEVYFx0-D3rCe_hZ7vDLfC9KT_9Lwexv07bJd7tTVQ&oe=62563C0D" alt="" data-position="center center" />
+								</a>
+								<div class="content">
+									<div class="inner">
+										<header class="major">
+											<h2> Want more from Sig Phi? Follow us on instagram @umdsigphi

```

Comment:
this title clips on smaller machines. Recommend putting "follow us..." as text, not the header. additionally, would be helpful to link @umdsigphi to the actual insta page